### PR TITLE
Add more UT test to resources_test

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
Add more UT test to resources_test
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by thockin** [comment]
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/60638)

**Comment by jianglingxia** [comment]
PTAL @deads2k  @vishh 
thanks very much!

**Comment by jianglingxia** [comment]
PTAL @deads2k @vishh
thanks very much!

File: pkg/quota/resources_test.go [hunk 1]
```diff
@@ -213,11 +225,16 @@ func TestContains(t *testing.T) {
 			b:        api.ResourceCPU,
 			expected: false,
 		},
-		"does-contain": {
+		"does-contain-cpu": {
 			a:        []api.ResourceName{api.ResourceMemory, api.ResourceCPU},
 			b:        api.ResourceCPU,
 			expected: true,
 		},
+		"does-contain-memory": {
```
Comment (deads2k) [inline pkg/quota/resources_test.go]: This case isn't needed. It's not testing any unique path, is it?

File: pkg/quota/resources_test.go [hunk 2]
```diff
@@ -192,6 +192,18 @@ func TestResourceNames(t *testing.T) {
 			},
 			expected: []api.ResourceName{api.ResourceMemory, api.ResourceCPU},
 		},
+		"values-one-cpu": {
```
Comment (deads2k) [inline pkg/quota/resources_test.go]: Link to the unique branch in the code this is testing?

File: pkg/quota/resources_test.go [hunk 3]
```diff
@@ -266,9 +283,23 @@ func TestIsZero(t *testing.T) {
 			},
 			expected: true,
 		},
-		"non-zero": {
+		"non-zero-both": {
```
Comment (deads2k) [inline pkg/quota/resources_test.go]: I don't see the unique branch this tests

Comment (jianglingxia) [reply pkg/quota/resources_test.go]: ok,If the unique branch case no need to test,I will close the pr!thanks!

**Pull request closed without merging**